frameworks/digital-outcomes-and-specialists-4/questions/brief-responses/essentialRequirements.yml

```diff
@@ -5,16 +5,13 @@ question_advice: |
 
   You’ll need to meet or exceed their requirements to get through to the next stage.
 
-  <h3>Evidence structure</h3>
-  <div class="explanation-list">
-    <p class="lead">When you write your evidence, you should be specific about:</p>
+  ## Evidence structure
```

Comment:
Did you mean to make this an h2 instead of an h3?

Reply:
Yup - there's no `<h2>` on that page.